### PR TITLE
various changes in docu

### PR DESCRIPTION
**Pull request by wdecker**

### CODE REVIEW

File: docs/src/CommutativeAlgebra/ca_affine_algebras.md [hunk 1]
```diff
@@ -114,6 +115,14 @@ degree(A)
...
 ## Ideals in Affine Algebras
 
+### Constructors
+
+```@docs
+ideal(g::Array{T, 1}) where {T <: MPolyQuoElem}
+```
+```@docs
+ ideal(Rx::MPolyQuo, g::Array{<:Any, 1})
+```
```
Comment (rfourquet) [inline docs/src/CommutativeAlgebra/ca_affine_algebras.md:125]: Just for your information, multiple signatures can be put in a single `@docs` block, e.g.
````suggestion
```@docs
ideal(g::Array{T, 1}) where {T <: MPolyQuoElem}
ideal(Rx::MPolyQuo, g::Array{<:Any, 1})
```
````

File: src/Rings/MPolyQuo.jl [hunk 2]
```diff
@@ -104,6 +104,31 @@ function gens(a::MPolyQuoIdeal)
...
+Create the ideal generated by the elements in `g`. 
+"""
+function ideal(g::Array{T, 1}) where {T <: MPolyQuoElem}
+  @assert length(g) > 0
+  @assert all(x->parent(x) == parent(g[1]), g)
+  return MPolyQuoIdeal(g)
+end
+
+@doc Markdown.doc"""
+    ideal(A::MPolyQuo, g::Array{<:Any, 1})
```
Comment (rfourquet) [inline src/Rings/MPolyQuo.jl:121]:
```suggestion
    ideal(A::MPolyQuo, g::Vector)
```

File: src/Rings/mpoly-ideals.jl [hunk 3]
```diff
@@ -28,9 +28,9 @@ function ideal(g::Array{T, 1}) where {T <: MPolyElem}
 end
 
 @doc Markdown.doc"""
-    ideal(Rx::MPolyRing, g::Array{<:Any, 1})
+    ideal(R::MPolyRing, g::Array{<:Any, 1})
```
Comment (rfourquet) [inline src/Rings/mpoly-ideals.jl:31]:
```suggestion
    ideal(R::MPolyRing, g::Vector)
```

Comment (wdecker) [reply src/Rings/mpoly-ideals.jl:31]: @rfourquet: Thanks, Wolfram

**Pull request merged**